tests/test_resources/run_properties.yml

```diff
@@ -0,0 +1,13 @@
+build:
+  run:
+    id: !ENV ${BUILD_ID:1}
+    run_url: !ENV ${RUN_DISPLAY_URL:http://localhost:3000/}
+    change_url: !ENV ${GIT_URL:http://localhost:3000/}
+    tests_url: !ENV ${RUN_TESTS_DISPLAY_URL:http://localhost:3000/}
+  parameters:
+    deploy_target: !ENV ${deployTarget:PullRequest}
+  versioning:
+    revision: !ENV ${GIT_COMMIT:hash}
+    branch: !ENV ${CHANGE_BRANCH}
+    pr_number: Null
+    tag: Null
```

Comment:
our `pyaml-env` should replace empty fields `!ENV ${CHANGE_BRANCH} ` with `None`. You can also leave out the field altogether to test this scenenario